interview.md

```diff
@@ -58,11 +111,34 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think the creator would get to spend a significant amount of time thinking
+about what would be most intuitive for a chemist to create code with. I am
+not sure how complex the visualization creation would be, but I would assume
+that we could use aspects of a general purpose language to create these
+visualizations. Outside of the visualization aspect, I do not think that there
+would be systems aspects because the rest of the project would be focused
+on creating the most intuitive language design for chemists.
+
```

Comment:
This idea definitely requires a lot more visualization which in turn may mean crafting some kind of GUI, especially if this is targeted towards chemists with minimal CS knowledge. This may be tedious in its own way, but you could definitely take inspiration from already existing systems to ease the load.